x-pack/legacy/plugins/code/server/server_options.ts

```diff
@@ -21,6 +21,9 @@ export interface SecurityOptions {
   installNodeDependency: boolean;
   gitHostWhitelist: string[];
   gitProtocolWhitelist: string[];
+  enableGitCertCheck: boolean;
```

Comment:
This option item has been removed in Yulong's native git PR, You probably bring in back when resolving conflicts. You probably need to create a new PR to fix this.

Also, not your fault, but can you delete this item in the `code/server/config.ts` as well? there might be some discrepancy between `master` and `7.x`.